README.md

```diff
@@ -131,7 +143,18 @@ Let's say the Posts in the previous question are available when you visit `http:
   3. If Step 1 is unsuccessful, print an error message to the console
 
 ```js
-// Your answer goes here...
+angular
+  .module('blog', [
+    'ngResource'
+  ])
+  .factory( 'postFactory', [
+    '$resource',
+    PostFactoryFunction
+  ])
+
+  function PostFactoryFunction($resource) {
+    return $resource("http://localhost:3000/posts.json")
+  }
 ```
 
```

Comment:
This is great if we wanted to do the request via Angular, but the question is asking for jQuery!